config/locales/en.yml

```diff
@@ -37,3 +37,10 @@ en:
           label: Yes, I've been tested and I do not have coronavirus
         option_no:
           label: No, I have not been tested
+    addiction:
+      title: Do you have a history of addiction?
+      options:
+        option_yes:
+          label: Yes, I’ve had problems with drug addiction or alcohol addiction (alcoholism)
+        option_no:
+          label: "No"
```

Comment:
Can we be consistent about the use of quotes or not in these options? Some have them, some don't, this one is inconsistent in itself...

Reply:
My preference is to not have them, but "No" is a reserved word, so needs the quotes.  Maybe we should just quote everything?